doc/source/security/encrypt-traffic.rst

```diff
@@ -12,6 +12,52 @@ This includes traffic between client applications and cluster nodes,
    :local:
    :depth: 1
 
+There are two approaches to configure encryption: automatic and manual.
+
+.. _ssl-auto-conf:
+
+SSL Automatic Configuration
+===========================
+
+|PXC| includes the :variable:`pxc-encrypt-cluster-traffic` variable
+that enables automatic configuration of SSL encryption.
+By default, it is disabled, meaning that you need to configure SSL manually
+if you want to encrypt SST and IST traffic, and internal communication.
+
+This variable cannot be changed on the command line during runtime.
+To enable automatic configuration of SSL encryption,
+set ``pxc-encrypt-cluster-traffic=ON`` in the the ``[mysqld]`` section of
+the :file:`my.cnf` file, and restart the cluster.
+
+Automatic configuration of the SSL encryption needs key and certificate files.
```

Comment:
Starting 5.7 key and cert are auto-generated by MySQL so this mode is like single step option. Let's keep it that simple.

It is like by default it would use the auto-generate cert but if user want user can override them by specifying the said options under mysqld section.